src/plasmapy/formulary/speeds.py

```diff
@@ -195,8 +196,8 @@ def ion_sound_speed(
     ion: ParticleLike,
     n_e: u.Quantity[u.m**-3] = None,
     k: u.Quantity[u.m**-1] = None,
-    gamma_e=1,
-    gamma_i=3,
+    gamma_e: float = 1,
+    gamma_i: float = 3,
```

Comment:
📝 In a few places, I had to correct `autotyping` because it saw that the default for `gamma_e` is an `int`, but in actuality it can be an `int` or `float`.  When this is the case, the convention that `mypy` uses is to use `float` rather than `int | float`.